django/db/backends/base/schema.py

```diff
@@ -1014,11 +1014,15 @@ def create_fk_name(*args, **kwargs):
         )
 
     def _create_unique_sql(self, model, columns):
+
+        def create_unique_name(*args, **kwargs):
+            return self.quote_name(self._create_index_name(*args, **kwargs))
```

Comment:
Maybe I missed sth but why not add this inside the `_create_index_name()` :thinking: . Probably other indexes names are also affected.

Reply:
I've initially tried using `quote_name` in `_create_index_name()`, but such change breaks "indexes" tests.
Also it would require changing `_create_fk_sql()` and `_create_index_sql()`, so I decided to go for a smaller code change.
However, there are suspicious usages of `_create_index_name()` at lines 727 and 767:
https://github.com/django/django/pull/10417/files#diff-b69190ab88f6c5737b2562d94d7bf36bR727
https://github.com/django/django/pull/10417/files#diff-b69190ab88f6c5737b2562d94d7bf36bR767

Reply:
I've searched for other `_create_index_name() ` misuses, but didn't find any.